src/libraries/System.Globalization.Calendars/tests/Misc/MiscCalendars.cs

```diff
@@ -42,5 +42,17 @@ public static void JapaneseTest()
             DateTime dTest = jCal.ToDateTime(1, 1, 8, 0, 0, 0, 0, 4);
             Assert.Equal(dTest, new DateTime(1989, 1, 8));
         }
+        
+        [Theory]
+        [InlineData(@"Thg 1", 1)]
+        [InlineData(@"Thg 2", 2)]
+        [InlineData(@"Thg 10", 10)]
+        [InlineData(@"Thg 11", 11)]
+        [InlineData(@"Thg 12", 12)]
+        public void VietnameseTest_MatchAbbreviatedMonthName_ContinuesSearchAfterMatch(string monthAbberavition, int expectedMonthNum)
+        {
+            var vi = new System.Globalization.CultureInfo("vi-VN");
```

Comment:
Thanks again for adding the test. I suggest making the test immune from future culture data changes by just setting the abbreviated month names on the DateTimeFormatInfo object of the culture. Something like:

```C#
var dtfi = new System.Globalization.CultureInfo("vi-VN").DateTimeFormat;
dtfi.AbbreviatedMonthNames = new string[] { "Thg 1", "Thg 2", "Thg 3", "Thg 4", "Thg 5", "Thg 6", "Thg 7", "Thg 8", "Thg 9", "Thg 10", "Thg 11", "Thg 12", "Thg 13" }; // 13 months
Assert.Equal(expectedMonthNum, DateTime.ParseExact(monthAbberavition, "MMM", dtfi).Month);
```
and you can try parse month 13 too.

Reply:
Thank you @tarekgh for the suggestions!
I'm still tripping over with attempts to test 13th month - as it needs to be supported by the `dtfi.Calendar` and at the same time such calendar need to be supported by the culture.
I got some success with some culture-calendar combinations (e.g. ar-IL and HebrewCalendar), however the parsing then adjust the Month ordinal number within result.
Do you have any top of head suggestions how to test the 13th month? Otherwise I'd probably stick to just 12 (as number of months are not anyways influenced by this change)

Reply:
I am ok to stick with 12 months for now. The test is already validating the case we need to check.